source/features/extend-conversation-status-filters.tsx

```diff
@@ -1,80 +1,65 @@
-import React from 'dom-chef';
-import {$, $$} from 'select-dom';
-import CheckIcon from 'octicons-plain-react/Check';
-import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager.js';
 import SearchQuery from '../github-helpers/search-query.js';
+import observe from '../helpers/selector-observer.js';
 
-function addMergeLink(): void {
+function addMergeLink(lastLink: HTMLAnchorElement): void {
+	// It's shouldn't be added in issues list
 	if (!pageDetect.isPRList()) {
 		return;
 	}
 
+	const locationQuery = SearchQuery.from(location);
+	const isMerged = locationQuery.includes('is:merged');
+	const isUnmerged = locationQuery.includes('is:unmerged');
+
 	// The links in `.table-list-header-toggle` are either:
 	//   1 Open | 1 Closed
 	//   1 Total            // Apparently appears with is:merged/is:unmerged
-	for (const lastLink of $$('.table-list-header-toggle.states a:last-child')) {
-		const lastLinkQuery = SearchQuery.from(lastLink);
-
-		if (lastLinkQuery.includes('is:merged')) {
-			// It's a "Total" link for "is:merged"
-			lastLink.lastChild!.textContent = lastLink.lastChild!.textContent.replace('Total', 'Merged');
-			continue;
-		}
-
-		if (lastLinkQuery.includes('is:unmerged')) {
-			// It's a "Total" link for "is:unmerged"
-			lastLink.lastChild!.textContent = lastLink.lastChild!.textContent.replace('Total', 'Unmerged');
-			continue;
-		}
-
+	if (isMerged) {
+		// It's a "Total" link for "is:merged"
+		lastLink.lastChild!.textContent = lastLink.lastChild!.textContent.replace('Total', 'Merged');
```

Comment:
It removed my `return` after running fix.

Reply:
The suggestion included dropping `else`

- https://github.com/sindresorhus/eslint-plugin-unicorn/issues/16